samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/DataTransformationSkills/TransformOutputSkill/config.json

```diff
@@ -0,0 +1,39 @@
+{
+  "schema": 1,
+  "description": "Apply a specific format or transformation to context variables inside a plan. This method must be used each time a specific format (such as specific format of list, specific JSON formatting from source data...) or transformation (such as filtering, capitalization...) is required for a parameter of a function in the plan and if another formatting or transformation function is not called instead. If a doubt exists about does the output format a function corresponds to the required input of another one, this methods must be called",
+  "type": "completion",
+  "completion": {
+    "max_tokens": 1000,
+    "temperature": 0.0,
+    "top_p": 0.0,
+    "presence_penalty": 0.0,
+    "frequency_penalty": 0.0,
+    "stop_sequences": [
+      "[END FORMATTEDOUTPUT]"
+    ]
+  },
+  "input": {
+    "parameters": [
+      {
+        "name": "input",
+        "description": "User intent",
+        "defaultValue": ""
+      },
+      {
+        "name": "rationale",
+        "description": "Description of the specific format or transformation that must be applied to inputVariable to respect the requested format or transformation. Must be as precise as possible and can contain examples of the expected result if available",
+        "defaultValue": ""
+      },
+      {
+        "name": "outputFormat",
```

Comment:
When is output format determined? Is the intention that Planner will run this skill every time it generates a new plan to align inputs and outputs? Or is it meant to be a step in every plan? I'd like to understand more about execution and your test scenarios

Reply:
The skill would be a step to align output of a previous step to a next step. 
For example: Step 1 return a JSON

Reply:
This is awesome, have you been able to test it? Can you share some positive and negative (if any) outcomes of plans that uses this skill?

Reply:
I too would love to see some examples of its usage. Seems like a great candidate to be a core plugin (or at min a function that the planner makes available). I've re-opened 1138 to make sure this change is tracked (as well as another suggestion of yours re: rationale)